src/document.rs

```diff
@@ -2,23 +2,36 @@
 
 use crate::git::get_current_user;
```

Comment:
Probably worth splitting this file up so that the schema (mostly autogen'd) is in a separate file?

Reply:
Yeah, I think it's probably worth a new folder with the contents broken up into separate files.